tests/framework/e2e/e2e.go

```diff
@@ -48,14 +48,13 @@ func (e e2eRunner) BeforeTest(t testing.TB) {
 
 func (e e2eRunner) NewCluster(ctx context.Context, t testing.TB, opts ...config.ClusterOption) intf.Cluster {
 	cfg := config.NewClusterConfig(opts...)
-	e2eConfig := EtcdProcessClusterConfig{
-		InitialToken:               "new",
-		ClusterSize:                cfg.ClusterSize,
-		QuotaBackendBytes:          cfg.QuotaBackendBytes,
-		DisableStrictReconfigCheck: !cfg.StrictReconfigCheck,
-		AuthTokenOpts:              cfg.AuthToken,
-		SnapshotCount:              cfg.SnapshotCount,
-	}
+	e2eConfig := NewConfig(
+		WithClusterSize(cfg.ClusterSize),
+		WithQuotaBackendBytes(cfg.QuotaBackendBytes),
+		WithDisableStrictReconfigCheck(!cfg.StrictReconfigCheck),
+		WithAuthTokenOpts(cfg.AuthToken),
+		WithSnapshotCount(cfg.SnapshotCount),
+	)
```

Comment:
Either way can work, it seems that I can we can just keep it as it's for now.

Reply:
Removing anti-pattern-naming fields(`DisableStrictReconfigCheck` and `NoCN`) is part of the motivation. To achieve that, I think all initialisations of `EtcdProcessClusterConfig` should be replaced by `DefaultConfig()` or `NewConfig `.

```go
func DefaultConfig() *EtcdProcessClusterConfig {
	return &EtcdProcessClusterConfig{
		ClusterSize:         3,
		InitialToken:        "new",
		CN:                  true,
		StrictReconfigCheck: true,
	}
}

func NewConfig(opts ...EPClusterOption) *EtcdProcessClusterConfig {
	c := DefaultConfig()
	for _, opt := range opts {
		opt(c)
	}
	return c
}
```